snippets/kickstart_networking_setup.erb

```diff
@@ -19,6 +19,9 @@ BOOTPROTO="<%= dhcp ? 'dhcp' : 'none' -%>"
 <% unless dhcp -%>
 IPADDR="<%= @host.ip -%>"
 NETMASK="<%= subnet.mask -%>"
+<% if subnet.gateway.present? %>
```

Comment:
`There was an error rendering the Katello Kickstart Default template: The snippet 'kickstart_networking_setup' threw an error: undefined method 'present?' for String::Jail (String)`